lib/LanguageServerRename/Model/Exception/EmptyNewName.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Phpactor\Extension\LanguageServerRename\Model\Exception;
+
+class EmptyNewName extends \Exception
+{
+    public function __construct(string $message = "", int $code = 0, \Throwable $previous = null)
```

Comment:
can remove `$code` and `$previous` and the other instances below.

Reply:
I could probably rebase on master. It's a bit difficult because if I do I lose my rename while I am working on this :-) That is why I did links to all the classes that are involved. Can I help some other way for you to review it while I am working on it?

Reply:
Why not check it out independently and work on it as a separate project ... that's what I do.

Reply:
I might as well do that... okay.

Reply:
Why remove `$code` and `$previous`?

Reply:
why wouldn't you? they are unnecessary - why would you pass `$code`, when would you pass `$previous`? it opens up the API unnecessarily.

Reply:
Oh, I see. Just this exception you mean.